grails-app/services/de/iteratec/osm/result/ApplicationDashboardService.groovy

```diff
@@ -118,4 +126,29 @@ class ApplicationDashboardService {
 
     }
 
+    List<Map> getAllActivePagesAndMetrics(Long jobGroupId) {
+        List<Map> recentMetrics = getRecentMetricsForJobGroup(jobGroupId).collect {
+            it.projectedProperties.pageName = Page.findById(it.pageId).name
+            return it.projectedProperties
+        }
+
+        getPagesOfActiveJobs(jobGroupId).each {
+            Page page = it
```

Comment:
set the closure's params instead of reassigning the implicit param it, please.

Reply:
done.